src/cmds/release.rs

```diff
@@ -101,27 +107,54 @@ pub struct ReleaseArgs {
 #[async_trait]
 impl CliCommand for ReleaseArgs {
     async fn run(&self, _cli: &Cli) -> anyhow::Result<()> {
-        let git = GitCli::new(self.owner.clone(), self.repo.clone());
+        if self.ticket_source_owners.is_empty() || self.ticket_source_repos.is_empty() {
+            return Err(anyhow!("ticket_source_owners and ticket_source_repos must be provided"));
+        }
```

Comment:
This is not required, since these new fields in ReleaseArgs are not optional. Mandatory fields w/ validation already.